docs/docs/ref/html_tokens.md

```diff
@@ -4,30 +4,28 @@ package: ALS Library
 group: Web
 module: pxml
 predicates:
+- {sig: 'tokenize_file/2', desc: 'extracts all Tokens from the File'}
 - {sig: 'grab_html_tokens/2', desc: 'extracts all Tokens from the File'}
 - {sig: 'read_tokens/2', desc: 'extracts all Tokens from a stream'}
-- {sig: 'tokenize_file/2', desc: 'extracts all Tokens from the File'}
 ---
 ## FORMS
 
+`tokenize_file(File, Tokens)`
+
 `grab_html_tokens(Path, Tokens)`
 
 `read_tokens(S, Tokens)`
 
-`tokenize_file(File, Tokens)`
-
 ## DESCRIPTION
 
-**`grab_html_tokens/2`** Just like `tokenize_file/2`, but wraps read_tokens in unwind_protect  
```

Comment:
The code-quoting changes I made in cc424432488cb83dd388cdc79fdfe2cb84cc8b15 have been lost here.

Reply:
Fixed by c84f348.